snap/backend/src/utils/networkUtils.js

```diff
@@ -0,0 +1,19 @@
+import statisticsClient from '../services/statisticsClient.js';
+import stateManager from '../stateManager.js';
+
+const networkUtils = {
+	async switchNetwork({ state, requestParams }) {
```

Comment:
Can the switchNetwork() be used in the index.js to initialize the default mainnet?

Reply:
yes, you are right, I will apply for that.